### ISSUE TITLE
Backend

### ISSUE THREAD

**Issue by amillion3** [amillion3/boro-web-dev#1]
Does anyone have experience with backend stuff? Some of these ideas will require it.

SQL or NoSQL? Firebase or Gun may be just fine for testing and development DBs. Totally new territory for me.

**Comment by meprice** [comment]
I am not that familiar with SQL but I am sure if we all put our heads together and even tap into some of our external resources we can get something figured out.

**Comment by amillion3** [comment]
Yes, I think we'll need external help or a bit of time to figure it out. I met someone at the FCC Nashville event today that _may_ be willing to help out. She has worked with Firebase and sounded interested in possibly helping out. Good stuff!

**Comment by amillion3** [comment]
This seems like a solid resource:
https://zellwk.com/blog/crud-express-mongodb/
